packages/api/src/auth/verifiers/sha1Verifier.ts

```diff
@@ -0,0 +1,121 @@
+import { createHmac, timingSafeEqual } from 'crypto'
+
+import {
+  VerifyOptions,
+  WebhookVerificationError,
+  VERIFICATION_ERROR_MESSAGE,
+  DEFAULT_WEBHOOK_SECRET,
+} from './index'
+import type { WebhookVerifier } from './index'
+
+export interface Sha1Verifier extends WebhookVerifier {}
+
+function toNormalizedJsonString(payload: Record<string, unknown>) {
+  return JSON.stringify(payload).replace(/[^\\]\\u[\da-f]{4}/g, (s) => {
+    return s.substr(0, 3) + s.substr(3).toUpperCase()
+  })
+}
+
+/**
+ *
+ * createSignature
+ *
+ */
+const createSignature = ({
+  payload,
+  secret = DEFAULT_WEBHOOK_SECRET,
+  options,
+}: {
+  payload: string | Record<string, unknown>
+  secret: string
+  options?: VerifyOptions
+}): string => {
+  if (options) {
+    console.warn('VerifyOptions are invalid for the Sha1Verifier')
+  }
```

Comment:
Are these warnings here because we have a single interface with for verifying different cryptos?

My inclination is to suggest that we expose different functions for each type of crypto since that'll be the most flexible interface, but maybe there's a nice way to button this up with TypeScript so we don't even show them that options is required. 

Just thinking out loud, there's no need for this to be blocking.